### PR TITLE
Added Webhook endpoints to WebhookClient

### PR DESCRIPTION
**Pull request by LuckyNoS7evin**

### CODE REVIEW

File: src/Pipedrive.net/Models/Request/NewWebhook.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        public long? UserId { get; set; }
+
+        [JsonProperty("http_auth_user")]
+        public string HttpAuthUser { get; set; }
+
+        [JsonProperty("http_auth_password")]
+        public string HttpAuthPassword { get; set; }
+
+        [JsonIgnore]
+        private readonly List<string> _validEventActions = new List<string>
```
Comment (DavidRouyer) [inline src/Pipedrive.net/Models/Request/NewWebhook.cs]: Why not introduce a new enum `EventAction`? No need to have an extra list here and you could remove the check in the constructor

File: src/Pipedrive.net/Models/Request/NewWebhook.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            "added",
+            "updated",
+            "merged",
+            "deleted",
+            "*"
+        };
+
+        [JsonIgnore]
+        private readonly List<string> _validEventObjects = new List<string>
```
Comment (DavidRouyer) [inline src/Pipedrive.net/Models/Request/NewWebhook.cs]: Why not introduce a new enum `EventObject`? No need to have an extra list here and you could remove the check in the constructor

Comment (LuckyNoS7evin) [reply src/Pipedrive.net/Models/Request/NewWebhook.cs]: @DavidRouyer for both EventObject and EventAction, introducing an enum was an idea until I thought about how to cover "*" and how that would be serialized on the CreateWebhook request.

File: src/Pipedrive.net/Models/Response/Webhook.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        public string HttpAuthUser { get; set; }
+
+        [JsonProperty("http_auth_password")]
+        public string HttpAuthPassword { get; set; }
+
+        [JsonProperty("last_delivery_time")]
+        public DateTime LastDeliveryTime { get; set; }
+
+        [JsonProperty("last_http_status")]
+        public string LastHttpStatus { get; set; }
```
[the remark this answers is not in the document]
Comment (LuckyNoS7evin) [reply src/Pipedrive.net/Models/Response/Webhook.cs]: oh will fix

File: src/Pipedrive.net/Models/Request/NewWebhook.cs (hunk 2 above)
Comment (DavidRouyer) [reply src/Pipedrive.net/Models/Request/NewWebhook.cs]: Try this https://stackoverflow.com/a/19768223

**Comment by LuckyNoS7evin** [comment]
@DavidRouyer I think we are good on this now, managed to get enums sorted, I'm actually running this in a production environment now with no issues so far

**Comment by DavidRouyer** [comment]
LGTM, thank you @LuckyNoS7evin!

**Pull request merged**